src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -629,7 +628,6 @@ private void packageTestResults() {
                 out.write(screenData, 0, screenData.length);
                 out.closeEntry();
             }
-            out.close();
         } catch (IOException e) {
             log.error(e);
         }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this "close" call; closing the resource is handled automatically by the try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS4087)